resources/views/process.blade.php

```diff
@@ -0,0 +1,41 @@
+@extends('app')
+@section('content')
+
+<style type="text/css">
+    #concept-col {
```

Comment:
Yeah, I copied most of the page and changed the class in the html and forgot that.